5.4.rational_Harry.py

```diff
@@ -0,0 +1,38 @@
+# הארי רציונלי אבל לא נורא
+
+# install if not exist "bs4" need for html parser
+
+from bs4 import BeautifulSoup
+import re
+import os
+import bs4
+
+def change_name_in_folder_by_func(get_name_func , directory=None):
+    # iterate over files in
+    # that directory
+    for filename in os.scandir(directory):
+        if filename.is_file():
+            new_name = get_name_func(filename.path)
+
+            os.rename(filename.path, os.path.join(directory, new_name))
+
+
+# get name from  of harry potter chapter from title
```

Comment:
Write documentation in the right format python - https://realpython.com/documenting-python-code/